src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -926,7 +1017,9 @@ class HttpGoogleServicesDAO(
       } match {
         case Success(response) =>
           logger.debug(GoogleRequest(request.getRequestMethod, request.buildHttpRequestUrl().toString, payload, System.currentTimeMillis() - start, Option(response.getStatusCode), None).toJson(GoogleRequestFormat).compactPrint)
-          response.parseAs(request.getResponseClass)
+          val result = processUnparsedResponse(response)
+          response.disconnect()
```

Comment:
We weren't previously doing this in `executeGoogleRequest`, but the JavaDoc for `AbstractGoogleClientRequest.executeUnparsed()` says it's up to the caller to call `disconnect()`.